docs/source/reference/howto/wipe_persistent_data.rst

```diff
@@ -134,4 +134,5 @@ Issues with check_services
     
      $ cchq monolith service kafka stop
      $ rm -rf /var/lib/zookeeper/*
+     $ rm -rf /opt/data/kafka/data/*
```

Comment:
> don't love that I hard code the paths here

Would it be worth adding a comment that the paths may need to be adjusted, depending on how Kafka is configured?

Reply:
Yeah probably: https://github.com/dimagi/commcare-cloud/pull/6221